tests/flatc/flatc_cpp_tests.py

```diff
@@ -249,6 +249,81 @@ def KeepPrefixIncludePrefixoutPath_SuperDirectoryReference(self):
             [
                 '#include "generated/baz/baz_generated.h"',
                 '#include "generated/foo_generated.h"',
-            ],
-            unlink=False,
+            ]
+        )
+
+    def ScopedEnumsDefault(self):
```

Comment:
These are fine, but probably would be better as a c++ unit test.

The tests in this file are more testing flags that interact with multiple files/includes statements that are harder to test in a small unit test.

Reply:
I have removed the unit tests because I could not find a way to test them.  The problem is that the test for --no-emit-min-max-enum-values would need to check if MIN and MAX symbols are not defined.  I tried various things with the preprocessor and std::enable_if, but the compiler always errors with "symbol not defined."  MIN and MAX do not appear anywhere else in the *_generated.h file so I cannot check to see if they appear in the array of strings of enumeration values that are generated.  I could try to have the C++ code do like the previous Python code and grep the *_generated.h header for MIN and MAX, but I thought I should check first.  If you know of a way to test, please let me know.  I am happy to write the tests.